tap_taboola/__init__.py

```diff
@@ -294,6 +345,10 @@ def do_sync(args):
         username=config.get('username'),
         password=config.get('password'))
 
+    singer.write_schema('account',
```

Comment:
Stream is named "accounts" 
```suggestion
    singer.write_schema('accounts',
```